extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,8 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+AMP.extension('amp-share-tracking', AMP => {
```

Comment:
is that first arg new?

i would have to to do:

```js
(function(AMP) {

}(AMP));
```
to mimic correctly instead of assuming global (which should also be ok)